src/panels/lovelace/cards/hui-sensor-card.ts

```diff
@@ -1,52 +1,55 @@
-import { LitElement, html, svg } from "@polymer/lit-element";
+import {
+  html,
+  LitElement,
+  PropertyDeclarations,
+  svg,
+} from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+import { until } from "lit-html/directives/until";
 
-import "../../../components/ha-card";
-import "../../../components/ha-icon";
+import { LovelaceCard } from "../types";
+import { LovelaceCardConfig } from "../../../data/lovelace";
+import { HomeAssistant } from "../../../types";
+import { fireEvent } from "../../../common/dom/fire_event";
 
 import computeStateName from "../../../common/entity/compute_state_name";
 import stateIcon from "../../../common/entity/state_icon";
 
-import EventsMixin from "../../../mixins/events-mixin";
-
-class HuiSensorCard extends EventsMixin(LitElement) {
-  set hass(hass) {
-    this._hass = hass;
-    const entity = hass.states[this._config.entity];
-    if (entity && this._entity !== entity) {
-      this._entity = entity;
-      if (
-        this._config.graph !== "none" &&
-        entity.attributes.unit_of_measurement
-      ) {
-        this._getHistory();
-      }
-    }
-  }
+import "../../../components/ha-card";
+import "../../../components/ha-icon";
+
+interface Config extends LovelaceCardConfig {
+  entity: string;
+  name?: string;
+  icon?: string;
+  graph?: string;
+  unit?: string;
+  detail?: number;
+  hours_to_show?: number;
+}
 
-  static get properties() {
+class HuiSensorCard extends LitElement implements LovelaceCard {
+  public hass?: HomeAssistant;
+  private _config?: Config;
+
+  static get properties(): PropertyDeclarations {
```

Comment:
We want to render on updates to `hass`, don't we now that you've removed `shouldUpdate`?

Reply:
Good catch